Больше деталей

jiykka opened the pull request:
---
:mortar_board: [Больше деталей](https://up.htmlacademy.ru/javascript-individual/1/user/2054473/tasks/8)

:boom: https://htmlacademy-javascript.github.io/2054473-kekstagram-1/3/

jiykka committed:
Main

jiykka committed:
Mainer

saimon395 commented on index.html:
```diff
@@ -16,7 +17,8 @@
       <h2 class="img-filters__title  visually-hidden">Фильтр фотографий</h2>
       <form class="img-filters__form" action="index.html" method="get" autocomplete="off">
-        <button type=button class="img-filters__button  img-filters__button--active" id="filter-default">По умолчанию</button>
+        <button type=button class="img-filters__button  img-filters__button--active" id="filter-default">По
+          умолчанию</button>
```
а зачем разметку правил ?

saimon395 commented on index.html:
```diff
@@ -227,10 +243,11 @@ <h2 class="success__title">Изображение успешно загруже
     <div class="img-upload__message  img-upload__message--loading">Загружаем...</div>
   </template>
-
+  <script src="js/main.js"></script>
```
адреса лучше относительнве использовать (./ в начале добавь)

jiykka committed:
Corrects

Разметка при сохранении выстраивается так. Отключил длину символов на строку - в след раз не будет по идее изменятся разметка))

saimon395 committed:
Merge branch 'master' into module4-task1
